### ISSUE TITLE
About accuracy in a sentences ?

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [lamhoangtung/LineHTR#4]
I used your code which you talked achieved is 0.84 (CER). But when I run yes CER is 0.81 ~ 0.84 but the sentence error rate is very low and appears overfitting on your model? You can expand my problem ??? 
Thank you for reading 
Thai Hoc

**Comment by lamhoangtung** [comment]
Hi, BiLTSM with CTC with many epochs can overfit easily. In order to train a more general OCR:
- Use as much as data as you can, apply image augmentation
- Try to pretrained the model on a synthesis dataset and then fine tune it on your smaller real dataset.

Hope this could help ;)

**Comment by lamhoangtung** [comment]
@NguyenThaiHoc1 May I close this issue ?
